lib/timers.js

```diff
@@ -151,11 +151,15 @@ exports._unrefActive = function(item) {
 // Appends a timer onto the end of an existing timers list, or creates a new
 // TimerWrap backed list if one does not already exist for the specified timeout
 // duration.
-function insert(item, unrefed) {
+function insert(item, unrefed, start) {
   const msecs = item._idleTimeout;
   if (msecs < 0 || msecs === undefined) return;
 
-  item._idleStart = TimerWrap.now();
+  if (start) {
```

Comment:
Because we know that `start` will always be an integer, this is probably more performant as `if (start !== 0)`.